data/virt_autotest/host_unattended_installation_files/autoyast/dev_host_15.xml.ep

```diff
@@ -58,8 +58,22 @@
       <!-- For some special beremetal machines, such as unreal2/3, their serial console:
       SERIALDEV='ttyS2', XEN_SERIAL_CONSOLE="com1=115200,8n1,0x3e8,5 console=com1" -->
       <xen_kernel_append><%= defined $bmwqemu::vars{"XEN_SERIAL_CONSOLE"} ? $bmwqemu::vars{"XEN_SERIAL_CONSOLE"} : "console=com2,115200" %> vga=gfx-1024x768x16 loglvl=all guest_loglvl=all sync_console</xen_kernel_append>
-      % } else {
-      <append>splash=silent console=<%= $get_var->('SERIALDEV') %>,115200 console=tty loglevel=5 <%= $check_var->('AMD', '1') ? "" : "intel_iommu=on" %> <%= defined $bmwqemu::vars{"OPT_KERNEL_PARAMS"} ? $bmwqemu::vars{"OPT_KERNEL_PARAMS"} : "" %></append>
+      % }
+      % if ($check_var->('SYSTEM_ROLE', 'kvm')) {
+        % if ($check_var->('AMD', '1')) {
+          % if ($check_var->('VIRT_SEV_ES_GUEST_INSTALL', '1')) {
+            <!-- Do not add mem_encrypt=on option on 15-SP6 KVM host due to bsc#1224107 -->
+            % if ($check_var->('VERSION_TO_INSTALL', '15-SP6') or ($check_var->('VERSION', '15-SP6') and !$get_var->('VERSION_TO_INSTALL'))) { 
```

Comment:
Actually `$check_var->('VERSION', '15-SP6')` is enough because `VERSION` has been set to the actual installed SLE version on host in `main.pm`. This line is redundant but not wrong.

Reply:
I will take this into consideration.

Reply:
Done